lib/src/clojure_lsp/db.clj

```diff
@@ -12,11 +12,10 @@
 
 (def initial-db {:documents {}
                  :processing-changes #{}
-                 :dep-graph {}
-                 :file-meta {}})
+                 :dep-graph {}})
 (defonce db* (atom initial-db))
 
-(def version 5)
+(def version 6)
```

Comment:
Do we really need to bump the version if we are removing `file-meta`? we don't need to care if existing loaded db would have that unnecessary entry, right?

Reply:
We need to re-index the external `analysis` by URI.